.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Ignore this file?

Reply:
I removed it from gitignore on purpose - user-specific settings should be in user config, and this one is workspace-specific, and sets the appropriate knobs that everyone should be using (e.g. run flake8).